python/arcticdb/version_store/_store.py

```diff
@@ -250,14 +249,22 @@ def create_store_from_lib_config(cls, lib_cfg, env, open_mode=OpenMode.DELETE):
         lib = cls.create_lib_from_lib_config(lib_cfg, env, open_mode)
         return cls(library=lib, lib_cfg=lib_cfg, env=env, open_mode=open_mode)
 
-    @classmethod
-    def create_store_from_config(
-        cls, cfg, env, lib_name, open_mode=OpenMode.DELETE, encoding_version=EncodingVersion.V1
+    @staticmethod
+    def create_library_config(
```

Comment:
This makes `arctic.py` much clearer now. There was no need to create a temporary Library just to get its config and delete it.